frontend/scripts/yarn-licenses.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+# 1. Install yarn
+npm install -D yarn
+
+# 2. Set up yarn: It will convert from package.json to yarn.lock
+npx yarn install
+
+# 3. List all packages with their license
+# npx yarn licenses list | less
+
+# # 4. Summarize licenses
+# yarn licenses list | grep License | sort -u | cut -d: -f2
```

Comment:
Should these two lines be commented out or removed?

Reply:
Sounds good, I moved them to the bottom for usage reference.